CHANGELOG.md

```diff
@@ -1,6 +1,26 @@
 Change Log
 ==========
 
+## Version 0.22
+
+_2019-12-9_
+
+### Kotlin
+
+ * BREAKING: Safer BackStackScreen construction. (#809)
+ * BREAKING: Pass ContainerHints via setContentWorkflow. (#808)
+ * Add variance to StatelessWorkflow type parameters. (#790)
+ * BREAKING CHANGE: ViewRegistry is now a ContainerHint. (#770)
+ * Make `lifecyclOrNull` public. (#747)
+ * Give Worker.doesSameWorkAs a default implementation that just compares by concrete type. (#746)
+ * BREAKING: Remove deprecated `testRender` API. (#743)
```

Comment:
Nit: Organize all the breaking changes under a sub-header? We've used that sort of formatting in the past.